samples/source-control/README.md

```diff
@@ -0,0 +1,24 @@
+source-control
+==============
+
+A project that depend on external sources:
+
+ 1. [external/](./external) implements the main algorithm to compute the answer to the ultimate question of Life, the Universe and Everything
+ 2. [sample/](./sample) implements the command line interface
+
+This sample uses a local Git repository, a realistic example would use a remote one.
+
+**Run `cd external && ./gradlew generateGitRepo` first** to generate the local repository.
+
+Then run with:
+
+    cd sample
+    ./gradlew run
+
+And check compilation dependencies with:
+
+    cd sample
+    ./gradlew dependencies --configuration compileClasspath
+
+See [sample/settings.gradle.kts](./sample/settings.gradle.kts).
+
```

Comment:
This really doesn't help explain what this is demoing.

I don't really understand what you are trying to demo here. Composite builds? Something else? Can you add some more details to this readme to explain?

Reply:
Done, see https://github.com/gradle/gradle-native/issues/42

Reply:
Thanks, I had no idea that this was related to building native code.